loops_lab.js

```diff
@@ -0,0 +1,316 @@
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+let num = 10;
+
+while (num >= 1){
+  console.log(num, "while loop");
+  num --;
+}
+
+for (num = 10; num >= 1; num --){
+  console.log(num, "for loop");
+}
+
+
+
+
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+let max = 100;
+
+let num2 = 0;
+while (num2 <= max){
+num2 % 2 !== 0
+? console.log(num2, "is odd while loop")
+: console.log(num2, 'is even while loop');
+num2 ++;
+}
+
+for (i = 1; i <= max; i ++){
+
+   i%2 !== 0
+   ? console.log(i, "is odd for loop")
+   : console.log(i, "is even for loop");
+ }
+
+
+
+
+
+
+
+
+
+
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+ let x = 12;
+ let num3 = 0;
+ let multiple = 9;
+
+ while (num3 <= x){
+   console.log(num3 * multiple);
+   num3 ++;
+ }
+
+
+
+
+
+
+
+
+
+//   _Bonus_ think of another way to solve it.
+//     <details>
+//       <summary>
+//         Hint
+//       </summary>
+//       Find the final number and increment the loop by 9.
+//     </details>
+//
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+
+let num4 = 0;
+
+ while(num4 < 100){
+   num4 +=5;
+   if (num4 % 5 === 0 && num4 % 10 !== 0 ){
+   console.log(num4);
+ }
+}
+
+
+
+
+
+
+
+
+
+
+// 5. Without running/executing your code, how many times will the loop below run? Explain why.
+//
+//  ```js
+// // let i = 5;
+// //
+// // while (i > 3) {
+// //     i += 1
+// // }
+// //
+// // // Your explanation here
+// this loop is an infinite loop. "i" is already greater then 3 and we're continually adding 1 to "i".
+//  ```
+
+
+// 6. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions.
+//For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+
+let num4 = 0;
+
+while (num4 <= 100){
+
+
```

Comment:
Minor spacing issue. Make sure to make your code look pretty and readable for other users. Proper spacing and indentation do the job perfectly.